### PR TITLE
Added Uzbek translation

### PR DESCRIPTION
**Pull request by nfzv**

### CODE REVIEW

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]
please add the language in `kibbeh` as well.
also don't forget to add the language keys for kibbeh in `kibbeh/src/ui/LanguageSelector` and for kofta in `kofta/src/app/components/LanguageSelector` *(even tho kofta won't be deployed anytime soon, but better safe than sorry)*

File: kofta/src/app/components/LanguageSelector.tsx [hunk 1]
```diff
@@ -65,6 +65,7 @@ export const LanguageSelector: React.FC<LanguageSelectorProps> = ({
     { value: "tr", label: "🇹🇷 Türkçe" }, // Turkish
     { value: "uk", label: "🇺🇦 Українська" }, // Ukrainian
     { value: "ur", label: "🇵🇰 اردو" }, // Urdu
+    { value: "uz", label: "🇺🇿 Oʻzbek" }, // Uzbek
```
Comment (amitojsingh366) [inline kofta/src/app/components/LanguageSelector.tsx:68]: add it to kibbeh's lang selector aswell

**Review by amitojsingh366 — CHANGES_REQUESTED** [review]

File: kofta/src/app/components/LanguageSelector.tsx (hunk 1 above)
Comment (nfzv) [reply kofta/src/app/components/LanguageSelector.tsx:68]: ` { value: "uz", label: "🇺🇿 Oʻzbek" }, // Uzbek` line was already added to kibbeh/src/ui/LanguageSelector.tsx as well.
Please let me know if I should add anything else. Thanks!

Comment (amitojsingh366) [reply kofta/src/app/components/LanguageSelector.tsx:68]: oh my bad i didnt notice. looks good

**Review by amitojsingh366 — APPROVED** [review]

**Pull request merged**